bentoml/artifact/__init__.py

```diff
@@ -33,6 +33,7 @@
 from bentoml.frameworks.spacy import SpacyModelArtifact
 from bentoml.frameworks.tensorflow import TensorflowSavedModelArtifact
 from bentoml.frameworks.xgboost import XgboostModelArtifact
+from bentoml.frameworks.transformers import TransformersModelArtifact
```

Comment:
The imports here in this file are for backward-compatible purpose only, it is ok to not add it here. Removing it from here should also help with the unit tests that are falling, due to missing the dependency in the CI environment.

Reply:
Removed in the new commit